example/azure-test-garden/README.md

```diff
@@ -0,0 +1,530 @@
+This example demonstrates how to install gardener into an existing AKS
+cluster. We'll use a single cluster for the sake of simplicity to host
+both gardener and a seed to the same cluster.
+
+Please note that this example is to provide you an example insallation
```

Comment:
Typo: `insallation` -> `installation`. 
Also, a little bit of tautology. Can you change it to `this example installation ...`.